src/main/scala/com/cloudera/sparkts/models/RegressionARIMA.scala

```diff
@@ -0,0 +1,198 @@
+package com.cloudera.sparkts.models
+
+import breeze.linalg
+import breeze.linalg.{DenseMatrix, DenseVector}
+import com.cloudera.sparkts.stats.TimeSeriesStatisticalTests._
+import org.apache.commons.math3.stat.regression.{OLSMultipleLinearRegression, SimpleRegression}
+
+import scala.language.postfixOps
+
+/**
+ * Created by baddar on 1/14/16.
+ * This object contains different methods to
+ */
+
+object RegressionARIMA {
+
+  val COCHRANE_ORCHUTT = 1;
+  val REG_ARMA = 2;
+
+  def fit(ts: linalg.Vector[Double], regressors: DenseMatrix[Double], method: Int, others: Any*): RegressionARIMAModel = {
```

Comment:
Call this  `fitModel` to be consistent with the other classes.  Also, for consistency, `method` should accept a String.  I think `optimizationArgs` would be a better name for `others`.